docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Below we highlight the top features, enhancements and bugfixes in this release, as well as relevant callouts for those upgrading from Grafana Mimir 2.1.
+The complete list of changes is recorded in the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir introduces new experimental support for ingesting out-of-order samples.
+  This support is configurable, with users able to set how far out-of-order Mimir will accept samples on a per-tenant basis.
+  **TODO: Link to documentation after https://github.com/grafana/mimir/pull/2183 is merged.**
+
+- **Backfill API**: Grafana Mimir 2.2 introduces new API for backfill. TODO: expand once mimirtool changes are in.
+
+- **Error messages**: We have made the error messages reported by Mimir more human readable and included error codes that can be easily searched for.
+  These error message improvements will continue in future releases.
+  You can find description of errors in [Grafana Mimir Runbooks / Codified errors]({{< relref "../operators-guide/mimir-runbooks/_index.md#codified-errors" >}}).
+
+- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each. Storage prefix can be configured using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
+
+## Upgrade considerations
+
+We have updated default values and some parameters in Grafana Mimir 2.2 to give users better out-of-the-box experience:
+
+- Message size limits for gRPC messages exchanged between internal Mimir components has been increased to 100 MiB from previous 4 MiB. This helps to avoid internal server errors when pushing or querying large data.
```

Comment:
```suggestion
- Message size limits for gRPC messages exchanged between internal Mimir components increased to 100 MiB from the previous 4 MiB. This helps to avoid internal server errors when pushing or querying large data.
```